connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java

```diff
@@ -234,6 +243,17 @@ Duration adminTimeout() {
         return props;
     }
 
+
+    Map<String, Object> targetConsumerConfig() {
+        Map<String, Object> props = new HashMap<>();
+        props.putAll(originalsWithPrefix(TARGET_CLUSTER_PREFIX));
+        props.keySet().retainAll(MirrorClientConfig.CLIENT_CONFIG_DEF.names());
+        props.putAll(originalsWithPrefix(CONSUMER_CLIENT_PREFIX));
+        props.put("enable.auto.commit", "false");
```

Comment:
Can we use the existing constants for the config names?

Reply:
done